webapp/src/ts/modals/edit-user/update-password.component.ts

```diff
@@ -76,6 +95,15 @@ export class UpdatePasswordComponent extends EditUserAbstract implements OnInit
     }
   }
 
+  private getLoginData (username, password) {
```

Comment:
I think this method should be moved to the new `UserLoginService`. 
The UpdatePassword component has no reason to know that our login endpoint accepts a "redirect" and a "locale" param. It would make more sense for it to pass the credentials alone, and the service should know how to build the request data. 

Does this make sense?

Reply:
You are right. Done, thanks.